test/system/070-build.bats

```diff
@@ -453,6 +453,40 @@ EOF
     run_podman rmi -a --force
 }
 
+@test "build with copy-from referencing the base image" {
+  _prefetch busybox
+  target=busybox-derived
+  target_mt=busybox-mt-derived
+  containerfile1=$tmpdir/Containerfile1
+    cat >$containerfile1 <<EOF
+FROM docker.io/library/busybox AS build
+RUN rm -f /bin/paste
+USER 1001
+COPY --from=docker.io/library/busybox /bin/paste /test/
```

Comment:
Could I encourage you to `s;docker.io/library/;quay.io/libpod/;` ?

Reply:
Yup cut and pasted from Buildah.